.github/workflows/release.yml

```diff
@@ -0,0 +1,257 @@
+name: release
+
+on:
+  push:
```

Comment:
not release trigger?

Reply:
Tag creates the release, marked as pre-release. Could do it the other way, but it's even more manual

Reply:
don't we want release to be a deliberate manual process? 
either way is fine

Reply:
It is a deliberate manual process, it's just one done via tag, push tag, rather than through the GH UI.